test/data/geojson/trees.geojson.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
could you move https://github.com/iTowns/itowns-sample-data ?

Reply:
> could you move https://github.com/iTowns/itowns-sample-data ?

I can use the existing test/data/geojson/points.geojson.json so I won't have to fetch data